nix/nginz.nix

```diff
@@ -0,0 +1,104 @@
+{ nginxModules
+, pkg-config
+, callPackage
+, stdenv
+, symlinkJoin
+, dockerTools
+, writeTextDir
+, runCommand
+, gnugrep
+, coreutils
+, which
+, nginx
+, inotify-tools
+, dumb-init
+, cacert
+, bashInteractive
+, openssl
+, openssl_1_1
+, lib
+, makeWrapper
+, writers
+}:
+let
+  zauth = callPackage ./pkgs/zauth { };
```

Comment:
We should have zauth already in `pkgs`, because we define it in `overlay.nix`.

Try passing zauth as a function argument in the head of the file.